tgui/packages/tgui/interfaces/HiveStatus.tsx

```diff
@@ -40,7 +40,7 @@ type InputPack = {
   user_ref: string;
   user_xeno: boolean;
   user_index: number;
-  user_queen: boolean;
+  user_ruler: boolean;
   user_watched_xeno: string;
   user_evolution: number;
   user_purchase_perms: boolean;
```

Comment:
This is nullable now
Should be a booleanlike too but whatever
```suggestion
  user_purchase_perms?: boolean;
```